src/nrepl/repl-start.ts

```diff
@@ -193,9 +193,9 @@ export async function startStandaloneRepl(
         tempDirUri,
         config.files[0]
     );
-    config.files.slice(1).forEach(async (file) => {
+    for (const file of config.files.slice(1)) {
         await openStoredDoc(storageUri, tempDirUri, file);
-    });
+    }
```

Comment:
https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-misused-promises.md

Reply:
I'm not sure I see a valid change here. Just removing that `);` would cause invalid code here.